services/common/src/constants/actionTypes.ts

```diff
@@ -1,5 +1,6 @@
 export const STORE_TEST_INFO = "STORE_TEST_INFO";
 export const STORE_BULK_STATIC_CONTENT = "STORE_BULK_STATIC_CONTENT";
+export const STORE_SYSTEM_FLAG = "STORE_SYSTEM_FLAG";
```

Comment:
I made it, it works, but then I ended up not using it anywhere. 😂 
Was going to use it to differentiate the project link on Core vs MS, but it did not make sense to import core & ms routes into common (at least not when I was working on it).